src/websocket-client.ts

```diff
@@ -373,7 +374,7 @@ export class WebsocketClient extends EventEmitter {
     this.logger.info('Websocket connection closed', { ...loggerCategory, wsKey});
 
     if (this.wsStore.getConnectionState(wsKey) !== READY_STATE_CLOSING) {
```

Comment:
```suggestion
    if (this.wsStore.getConnectionState(wsKey) !== READY_STATE_CLOSING && this.options.reconnectOnClose) {
```

You could do this instead. This ensures the state is reset and the "close" event is dispatched (for any implementations wanting to know when a ws closed, since a reconnect will no longer happen)

Reply:
https://github.com/kenchambers/ftx-api/commit/76817984fe9875f36f0389bd6776d720aa6663b6 @tiagosiebler 

nice catch!